Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#Step 1 - Update CentOS 7 system
+echo "Updating system"
+sudo yum -y install epel-release
+sudo yum -y update
+sudo yum install -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+
+#Install Docker
```

Comment:
remove comment.
Don't you think the echo is enough for those who read the code?